### PR TITLE
Update LiveSplit.AutoSplitters.xml

### PR DESCRIPTION
**Pull request by BurrowBird**
Updated my auto splitter to support loading time removal (game time).

If you are adding or modifying an Auto Splitter, please fill out this checklist:
- [✓] My Auto Splitter does not contain any malicious functionality and I'm entirely responsible for any damages myself. (Any kind of abuse in an Auto Splitter will result in an immediate ban.)
- [✓] I am not replacing an existing Auto Splitter by a different author, or I have received permission from the author to replace the existing Auto Splitter.
- [✓] The Auto Splitter has an open source license that allows anyone to fork and host it.

### CODE REVIEW

File: LiveSplit.AutoSplitters.xml [hunk 1]
```diff
@@ -13486,10 +13486,10 @@
             <Game>ShadowFlare</Game>
         </Games>
         <URLs>
-            <URL>https://raw.githubusercontent.com/BurrowBird/Splitters/master/SF_Auto_Splitter.asl</URL>
+            <URL>https://raw.githubusercontent.com/BurrowBird/Splitters/master/SF_Auto_Splitter_v7%202024-03-23.asl</URL>
```
Comment (just-ero) [inline LiveSplit.AutoSplitters.xml:13489]: Are you sure you want to include version information in the file name? You will have to PR this repo every time you rename the file.

Comment (BurrowBird) [reply LiveSplit.AutoSplitters.xml:13489]: My last update was a few years ago, but I guess not.

Comment (BurrowBird) [reply LiveSplit.AutoSplitters.xml:13489]: I guess I can leave the link as is and just update the splitter directly, since I can do that instead, right?

Comment (just-ero) [reply LiveSplit.AutoSplitters.xml:13489]: Sure.

Comment (BurrowBird) [reply LiveSplit.AutoSplitters.xml:13489]: I have no idea how to do that though. Cancel this one and create a new request?

Comment (just-ero) [reply LiveSplit.AutoSplitters.xml:13489]: Just edit the file on your branch again.

**Comment by just-ero** [comment]
Moved to #1714, I guess...

**Pull request closed without merging**